app/screens/channel/channel_post_list/index.js

```diff
@@ -27,7 +26,7 @@ function mapStateToProps(state) {
         deviceHeight: state.device.dimension.deviceHeight,
         postIds: getPostIdsInCurrentChannel(state),
         postVisibility: state.views.channel.postVisibility[channelId],
-        lastViewedAt: currentChannelMember && currentChannelMember.last_viewed_at,
+        lastViewedAt: state.views.channel.lastChannelViewTime[channelId],
```

Comment:
Why add a new reducer to track this? Why wouldn't the channel member's last_viewed_at be updated? We already have multiple ways of tracking when we we've viewed/read a channel, and I'm 4/5 against adding another way to do that.

Reply:
this is exactly the way is done on the webapp and the main reason for this ticket, as if we mark the channel as read the `new message indicator` will disappear.

Reply:
Oh right, I forgot about that line. This makes much more sense then

In that case, do we need to store a different value for each channel? What if we just stored a single timestamp for that line and then we change it every time you switch to a new channel?